### PR TITLE
5.4.0 tiup bump version

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -112,9 +113,9 @@ tiup update cluster
 
 > **注意：**
 >
-> 升级到 5.3 版本前，请确认已在 4.0 修改的参数在 5.3 版本中是兼容的，可参考 [TiKV 配置文件描述](/tikv-configuration-file.md)。
+> 升级到 5.4 版本前，请确认已在 4.0 修改的参数在 5.4 版本中是兼容的，可参考 [TiKV 配置文件描述](/tikv-configuration-file.md)。
 >
-> 以下 TiKV 参数在 TiDB v5.3 已废弃。如果在原集群配置过以下参数，需要通过 `edit-config` 编辑模式删除这些参数：
+> 以下 TiKV 参数在 TiDB v5.4 已废弃。如果在原集群配置过以下参数，需要通过 `edit-config` 编辑模式删除这些参数：
```
Comment (AstroProfundis) [inline upgrade-tidb-using-tiup.md:118]: 这个我觉得不应该改版本？用 5.3 及之后版本之类的说法更好些？

Comment (qqqdan) [reply upgrade-tidb-using-tiup.md:118]: 呃 这个貌似是历史传下来的，我看之前版本都每次发版都加一下。 @TomShawn 你这边有没有建议啊？
image

Comment (TomShawn) [reply upgrade-tidb-using-tiup.md:118]: 额，感觉是之前搞错了。如果这些配置项是在 v5.0 废弃的话，这里就不该 bump version 了
```suggestion
> 以下 TiKV 参数在 TiDB v5.0 已废弃。如果在原集群配置过以下参数，需要通过 `edit-config` 编辑模式删除这些参数：
```

**Review by AstroProfundis — APPROVED** [review]

**Review by seiya-annie — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**